app/src/test/java/com/vero/tasky/agenda/data/repository/EventRepositoryImplTest.kt

```diff
@@ -0,0 +1,213 @@
+        description = null,
+        remindAt = LocalDateTime.now(),
+        time = LocalDateTime.now(),
+        to = LocalDateTime.now(),
+        host = "",
+        isUserEventCreator = true,
+        attendees = fakeListOfAttendee,
+        photos = fakeListOfLocalPhotos
+    )
+
+    @Before
+    fun setUp() {
+        mockWebServer = MockWebServer()
+        okHttpClient = OkHttpClient.Builder()
+            .readTimeout(1, TimeUnit.SECONDS)
+            .callTimeout(1, TimeUnit.SECONDS)
+            .connectTimeout(1, TimeUnit.SECONDS)
+            .build()
+        api = Retrofit.Builder()
+            .addConverterFactory(MoshiConverterFactory.create())
+            .baseUrl(mockWebServer.url("/"))
+            .client(okHttpClient)
+            .build()
+            .create(EventApi::class.java)
+        eventDao = EventDaoFake()
+        alarmHandler = AlarmHandlerFake()
+        modifiedAgendaItemDao = ModifiedAgendaItemDaoFake()
+        eventRepository = EventRepositoryImpl(
+            alarmHandler = alarmHandler,
+            api = api,
+            eventDao = eventDao,
+            modifiedAgendaItemDao = modifiedAgendaItemDao,
+            saveEventRunner = mockk(relaxed = true),
+            multipartParser = mockk(relaxed = true) {
+                every { getMultipartPhotos(fakeListOfLocalPhotos) } returns listOf(
+                    MultipartBody.Part.createFormData("", "",)
+                )
+                every { getMultipartPhotos(emptyList()) } returns listOf()
+            }
+        )
+    }
+
+    @After
+    fun tearDown() {
+        mockWebServer.shutdown()
+    }
+
+    @Test
+    fun `Save event, one image was skipped`() = runTest {
+        val result = eventRepository.saveEvent(
+            event = event,
+            isGoing = true,
+            deletedPhotoKeys = emptyList(),
+            modificationType = ModificationType.CREATED
+        )
+        Truth.assertThat(result.getOrNull()?.countOfSkippedMessages).isEqualTo(1)
+    }
+
+    @Test
+    fun `Save event, event was saved successfully`() = runTest {
+        eventRepository.saveEvent(
+            event = event,
+            isGoing = true,
+            deletedPhotoKeys = emptyList(),
+            modificationType = ModificationType.CREATED
+        )
+        Truth.assertThat(eventDao.loadAllEvents().size).isEqualTo(1)
+    }
+
+    @Test
+    fun `Delete event, event was deleted successfully`() = runTest {
+        mockWebServer.enqueue(
+            MockResponse()
+                .setResponseCode(200)
+        )
+
+        eventRepository.saveEvent(
+            event = event,
+            isGoing = true,
+            deletedPhotoKeys = emptyList(),
+            modificationType = ModificationType.CREATED
+        )
+        eventRepository.deleteEvent(
+            event = event
+        )
+        Truth.assertThat(eventDao.loadAllEvents().size).isEqualTo(0)
+    }
+
+    @Test
+    fun `Delete event, event wasn't deleted`() = runTest {
```

Comment:
IMO a little bit confusing test name, since I would typically expect the name is `Action, expected outcome`, so the action would be deleting an event here and the expected outcome that the event wasn't deleted which doesn't make much sense to me 🤔